app/controllers/ApplicationController.scala

```diff
@@ -44,7 +45,9 @@ class ApplicationController @Inject()(
   failedIvIncompleteView: FailedIvIncompleteView,
   lockedOutView: LockedOutView,
   timeOutView: TimeOutView,
-  technicalIssuesView: TechnicalIssuesView)(
+  technicalIssuesView: TechnicalIssuesView,
+  val notFoundView: NotFoundView,
+  val errorView: ErrorView)(
```

Comment:
I think the better fix is to remove the RenderErrors trait, turn it into a class and DI it in wherever these two views (or RenderError functions) are required